test/extensions/filters/network/postgres_proxy/postgres_test_config.yaml

```diff
@@ -3,6 +3,7 @@ admin:
   - name: envoy.access_loggers.file
     typed_config:
       "@type": type.googleapis.com/envoy.extensions.access_loggers.file.v3.FileAccessLog
+      # im wondering if this should just be ""
```

Comment:
"/dev/null" maybe? Please add a TODO comment if you're not going to change it.

Reply:
apologies for delay on this - im not entirely sure best way to update - ill set to `/dev/null` and if that works lets go with that...

Reply:
it turns out that the test is actually expecting `{}` so reverted and removed my comment